recipes/au/all/patches/0001-v0.3.5-disable-gtest.patch

```diff
@@ -0,0 +1,58 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
Proposed a solution upstram, but their CMake is a bit odd so some feedback there is welcome ;)
https://github.com/aurora-opensource/au/pull/297

Reply:
Thank you for posting it to the upstream first!